### PR TITLE
Feature/1 project setup (resolves #1)

### LINKED ISSUE #1: Create a "Hello World" frontend

### PR DESCRIPTION
**Pull request by Krismix1**
resolves #1

**Changes introduced:**
- Set up Angular project
- Read hello world from backend

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+  "name": "linkedin-clone-frontend",
+  "version": "0.0.0",
```
Comment (bambooSocks) [inline package.json:3]: do we care about the version?

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Editor configuration, see https://editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
```
[the remark this answers is not in the document]
Comment (Krismix1) [reply .editorconfig:7]: 2 spaces is a common standard for CSS, HTML and TS.

File: package.json (hunk 1 above)
Comment (Krismix1) [reply package.json:3]: not yet.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,11 @@
...
+# LinkedIn Clone Frontend
+
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 9.1.6.
+
+## Development
+
+1. Install Node either using [nvm](https://github.com/nvm-sh/nvm) or directly. Check the `.nvmrc` for recommended version.
+2. Install [yarn](https://classic.yarnpkg.com/en/docs/install/).
+3. Install dependencies using `yarn install`.
+4. Run the dev server using `npm run start`.
```
[the remark this answers is not in the document]
Comment (Krismix1) [reply README.md]: fixed

[the remark this answers is not in the document]
Comment (bambooSocks) [reply README.md]: I appreciate

**Pull request merged**